### PR TITLE
Changed style of "PRs Welcome" badge

### PR DESCRIPTION
**Pull request by sl**
Changed the style of the "PRs Welcome" badge from flat to shields so that it matches the Travis build status badge.

### CODE REVIEW

**Comment by athityakumar** [comment]
@sl - Thanks for contributing. If you're interested,  can you also add 'Built with Ruby' and 'Built with love' badges?

**Comment by sl** [comment]
I don't think that there's a shields.io badge for Built with ruby / built with love. You'd need to design the icons for those yourself or if you know where I can grab the image for already existing ones, I'd be happy to toss them in with this PR. I would create the badges myself, but I'm an awful designer, haha.

**Comment by athityakumar** [comment]
Haha, no need to create new badges. Check the below links -

- Made with love : https://github.com/boennemann/badges
- Awesome : https://github.com/bevacqua/awesome-badges
- Made with Ruby : Seems like this one is down. No problem. 😄

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Color LS 
 
 [![Build Status](https://travis-ci.org/athityakumar/colorls.svg?branch=master)](https://travis-ci.org/athityakumar/colorls)
-[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=shields)](http://makeapullrequest.com)
 
 A Ruby script that colorizes the `ls` output with color and icons. Here are the screenshots of working example on an iTerm2
 terminal (Mac OS), `oh-my-zsh` with `powerlevel9k` theme and `powerline nerd-font + awesome-config` font.
```
Comment (athityakumar) [inline README.md]: @sl - If you're not done with adding the badges yet, I'd like to request one more change - can you also please add 'Solarized dark color theme' in line 7 after the font specification? 😄

Comment (sl) [inline README.md]: Sure!

**Comment by athityakumar** [comment]
Almost ready to merge. Just add the awesome badge from https://github.com/bevacqua/awesome-badges, and reorder the badges to make them look good on README.

**Comment by sl** [comment]
What order were you thinking? The nicest looking one I could come up with is 

On the first line: Made with ruby, Built with love
On the second line: Build: Passing, PRs Welcome, Awesome

But if there's an order you'd prefer, let me know

**Pull request merged**